File: src/main/java/baboon/industry/block/storage/entity/TileEntityBatboxMV.java
package baboon.industry.block.storage.entity;

import baboon.industry.IndustryConfig;

public class TileEntityBatboxMV extends TileEntityBatboxBase {

    public TileEntityBatboxMV() {
        setCapacity(IndustryConfig.cfg.getInt("Energy Values.mvBatteryStorage") * 3);
        setTransfer(IndustryConfig.cfg.getInt("Energy Values.mvIO"));
        setMaxReceive(IndustryConfig.cfg.getInt("Energy Values.mvIO"));
    }

    @Override
    public String getInvName() {
        return "IndustryBatboxMV";
    }
}

File: src/main/kotlin/turniplabs/industry/blocks/entities/batbox/TileEntityBatboxMV.kt
package turniplabs.industry.blocks.entities.batbox

import sunsetsatellite.sunsetutils.util.Connection
import sunsetsatellite.sunsetutils.util.Direction
import turniplabs.industry.IndustryConfig

class TileEntityBatboxMV() : TileEntityBatboxBase() {

    init {
        this.setCapacity(32768)
        this.setTransfer(IndustryConfig.cfg.getInt("Energy Values.mediumVoltage"))
        this.setMaxReceive(IndustryConfig.cfg.getInt("Energy Values.mediumVoltage"))

        for (dir in Direction.values())
            this.setConnection(dir, Connection.OUTPUT)

        this.setConnection(Direction.Y_POS, Connection.INPUT)
    }

    override fun getInvName(): String {
        return "BatboxMV"
    }
}